app/views/assignments/_filterable_assignment_roster_list.html.erb

```diff
@@ -1,29 +1,35 @@
+<<<<<<< Updated upstream
 <div class='TableObject'>
+=======
+<% if search_assignments_enabled? %>
+  <div class='TableObject'>
+>>>>>>> Stashed changes
```

Comment:
It looks like some merge conflicts from applying a stash still exist :joy:

Reply:
😞I'll fix it